tests/test_dask_image/test_ndinterp/test_affine_transformation.py

```diff
@@ -1,13 +1,33 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
 
+from packaging import version
+
+import dask
+import dask.array as da
+import numpy as np
 import pytest
+import scipy
+from scipy import ndimage
```

Comment:
This is a nitpick, but could we use `import scipy.ndimage` here instead of `from scipy import ndimage`?

I've started to do a bit of code style cleanup here https://github.com/dask/dask-image/pull/227, and `import scipy.ndimage` is what we'll try to use consistently across the codebase.